content/zh-cn/blog/_posts/2022-07-13-gateway-api-in-beta.md

```diff
@@ -0,0 +1,335 @@
+---
+layout: blog
+title: Kubernetes Gateway API 毕业至 Beta 版本
+date: 2022-07-13
+slug: gateway-api-graduates-to-beta
+canonicalUrl: https://gateway-api.sigs.k8s.io/blog/2022/graduating-to-beta/
```

Comment:
Not sure should the canonicalUrl of a Mandarin post be the English one or not.

CC @tengqm How do you think?

Reply:
I think it's helpful as a portal to the complete user guide.

Reply:
I don't have a strong opinion on this. Maybe linking to the English page is not a bad idea if we don't have other options.

Reply:
The `canonicalUrl` is for "duplicated pages".
https://developers.google.com/search/docs/advanced/crawling/consolidate-duplicate-urls

Not like `full_link` means "see also".

Reply:
I think it's better to remove it in localized page.
But since maybe it only affect robot behavior (e.g. SEO), it's fine for me if PR author think it's unnecessary to remove it.

Reply:
(Also CC @sftim FYI)

Reply:
I would not set a canonical URL. There is metadata to make the English version discoverable (IIRC), and the English version has that canonical link.

Reply:
ok, removed this line of canonicalUrl.